dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataImport.java

```diff
@@ -1169,10 +1169,8 @@ private static int displayChanges(List<BulkEditChange> changes, boolean changed)
      */
     private static boolean isAuthorityControlledField(String md)
     {
-        int pos = md.indexOf("[");
-        String mdf = (pos > -1 ? md.substring(0, pos) : md);
-        pos = md.indexOf(":");
-        mdf = (pos > -1 ? md.substring(pos+1) : md);
```

Comment:
The old 1175 is where the bug resides: it should substring the mdf string and NOT the original md string.